src/test/CMakeLists.txt

```diff
@@ -508,10 +508,10 @@ add_executable(ceph_test_admin_socket_output
   )
 target_link_libraries(ceph_test_admin_socket_output
   ceph-common)
-if(CMAKE_CXX_COMPILER_ID STREQUAL GNU)
+if(CMAKE_CXX_COMPILER_ID STREQUAL GNU OR CLANG_CXXLIB STREQUAL "libstdc++")
```

Comment:
thanks. but ... i think the library to be linked just depends on the C++ library we are using, so we should put
```cmake
if(CLANG_CXXLIB STREQUAL "libstdc++")
  # ...
elseif(CLANG_CXXLIB STREQUAL "libc++")
  # ...
endif()
```

what do you think?